codeSnippets/snippets/auth-basic-hash-table/build.gradle

```diff
@@ -0,0 +1,18 @@
+plugins {
+    id 'application'
+    id 'org.jetbrains.kotlin.jvm'
+}
+
+mainClassName = "io.ktor.server.netty.EngineMain"
+
+repositories {
+    mavenCentral()
+    maven { url "https://maven.pkg.jetbrains.space/public/p/kotlinx-html/maven" }
```

Comment:
What is this repository for?

Reply:
copy/paste from another example, removed